lib/error.js

```diff
@@ -540,30 +540,36 @@ AppError.messageRejected = (reason, reasonCode) => {
   })
 }
 
-AppError.emailComplaint = () => {
+AppError.emailComplaint = (bouncedAt) => {
   return new AppError({
     code: 400,
     error: 'Bad Request',
     errno: ERRNO.BOUNCE_COMPLAINT,
     message: 'Email account sent complaint'
+  }, {
+    bouncedAt
```

Comment:
tiny nit: maybe `lastBouncedAt` would make it clearer to consumsers exactly what this represents?